Shopify/cli
# Install cloudflared at runtime

### WHY are these changes introduced?
We were installing cloudflared on a postinstall script but that script was in JS, was blocking CLI installtion for some users who can't download cloudflare and slows down installation for others.




### WHAT is this pull request doing?
- Remove the post-install step
- Download cloudflare the first time you run `dev` (if necessary)



### How to test your changes?
- remove the cloudflare binary from `plugin-cloudflare/bin/`
- Run `dev`, it should reinstall the binary and work fine
- Run `dev` again, it should not install it this time and work fine.


### Post-release steps



### Measuring impact

How do we know this change was effective? Please choose one:


### Checklist
